### PR TITLE
Tweak some nutrition values

### PR DESCRIPTION
**Pull request by lbnesquik**
## About this pull request 
- Gives cream & Milk higher nutrition values 
- Make ethanol give at least 2 nutrition per unit as a failsafe for any drinks. It could be used to remove some values bloat as well.
- Indents some comments to look prettier.

## Why is this good for the game 
Milk and cream are acceptably thick in nutrition as it should be.
Code is slightly prettier to read
This makes drinks behave a tad more realistically without much cost. 

It could still use a more thorough pass but as you can see in this example, it isn't clear why some drinks give you some, and others give you zero.
image

### CODE REVIEW

**Comment by lbnesquik** [comment]
I moved nutrition straight to reagents, so we don't have that SAME var defined in two places.

**Review by MistakeNot4892 — CHANGES_REQUESTED** [review]
You'll need to move/redefine/unify some logic from `/datum/reagent/nutriment` and `/datum/reagent/ethanol` to `/datum/reagent`, specifically the adjust_nutrition calls:
````
/datum/reagent/ethanol/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
	if(issmall(M)) removed *= 2
	if(!(M.species.allergens & allergen_type))	//assuming it doesn't cause a horrible reaction, we get the nutrition effects
		M.adjust_nutrition(nutriment_factor * removed)
		
/datum/reagent/nutriment/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
	switch(alien)
		if(IS_DIONA) return
		if(IS_UNATHI) removed *= 0.5
	if(issmall(M)) removed *= 2 // Small bodymass, more effect from lower volume.
	if(!(M.species.allergens & allergen_type))	//assuming it doesn't cause a horrible reaction, we'll be ok!
		M.heal_organ_damage(0.5 * removed, 0)
		M.adjust_nutrition(nutriment_factor * removed)
		M.add_chemical_effect(CE_BLOODRESTORE, 4 * removed)

**Comment by lbnesquik** [comment]
I moved some things around and it should be good to go this time around.

**Comment by lbnesquik** [comment]
Was clearing branches, soz

File: code/modules/reagents/reagents/_reagents.dm [hunk 1]
```diff
@@ -186,13 +203,136 @@
...
+
+		if(halluci)
+			M.hallucination = max(M.hallucination, halluci*3)
+
+
+
 	return
 
 /datum/reagent/proc/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
+	if(nutriment_factor)
```
Comment (MistakeNot4892) [inline code/modules/reagents/reagents/_reagents.dm:274]: Huge amount of duplicate code here. Should really move it into a proc or just have affect_blood call affect_ingest.

Comment (lbnesquik) [reply code/modules/reagents/reagents/_reagents.dm:274]: Well i am just moving things around. A proper brush through to see what the deal is with those variables is a bit outside the scope of this PR

Comment (MistakeNot4892) [reply code/modules/reagents/reagents/_reagents.dm:274]: I'm not talking about the variables - you have added identical code to affect_ingest and affect_blood that could be moved into a single proc and called, reducing the amount of code you're adding/that has to be maintained.

Comment (lbnesquik) [reply code/modules/reagents/reagents/_reagents.dm:274]: I don't know how to say it politely but i really do not feel like putting 3/4 hours into making a proper, unified proc to handle this. 
And yes, while there is duplicate code, this is not an addition, given it was all pretty much here before, if only scattered across multiple files.

Comment (MistakeNot4892) [reply code/modules/reagents/reagents/_reagents.dm:274]: It's nearly identical code. It should take maybe ten minutes to split it out.
Someone else will need tonreview/approve if you don't want to do that as this level of copypaste is something that contributes to poor long term code quality and maintainability.

**Pull request closed without merging**